packages/postgres-database/src/simcore_postgres_database/models/products.py

```diff
@@ -36,6 +36,7 @@ class Vendor(TypedDict, total=False):
     url: str
     license_url: str  # Which are the license terms? (if applies)
     invitation_url: str  # How to request a trial invitation? (if applies)
+    landing_page: bool  # Landing page enabled
```

Comment:
MINOR:  i suggest to rename as  ``enable_landing_page`` or ``landing_page_enabled`` or ``has_landing_page`` ...clearly denote a boolean

Reply:
``has_landing_page`` 👌